### PR TITLE
Passcode Reset: Activity log#6702

### PR DESCRIPTION
**Pull request by shabeeb-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/6702

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/components/auth/account-settings/activity-log/ActivityLog.vue [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+    {
+      text: 'Subject',
+      align: 'left',
+      sortable: false,
+      value: 'action',
+      class: 'bold-header'
+    }
+  ]
+
+  //   {
```
Comment (sumesh-aot) [inline auth-web/src/components/auth/account-settings/activity-log/ActivityLog.vue:95]: Remove this if not needed

File: auth-web/src/routes/router.ts [hunk 2]
```diff
@@ -222,6 +224,14 @@ export function getRoutes (): RouteConfig[] {
           path: 'product-settings',
           name: 'product-settings',
           component: productPackage
+        },
+        {
+          path: 'activity-log',
+          name: 'activity-log',
+          component: activityLog,
+          meta: {
+            isPremiumOnly: true
```
Comment (sumesh-aot) [inline auth-web/src/routes/router.ts:233]: Don't we need to show activity logs to staff for all (including basic) accounts ?

**Review by sumesh-aot — COMMENTED** [review]
Just one comment about showing this menu, also please include some test cases

File: auth-web/src/components/auth/account-settings/activity-log/ActivityLog.vue [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+
+  public async loadActivityList (pageNumber?: number, itemsPerPage?: number) {
+    this.isDataLoading = true
+    const filterParams: ActivityLogFilterParams = {
+      pageNumber: pageNumber,
+      pageLimit: itemsPerPage,
+      orgId: this.currentOrganization.id
+    }
+
+    const resp:any = await this.getActivityLog(filterParams)
```
Comment (karthik-aot) [inline auth-web/src/components/auth/account-settings/activity-log/ActivityLog.vue:123]: do we need a try.. catch ?

**Review by karthik-aot — APPROVED** [review]

**Pull request merged**